common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -543,60 +547,66 @@ public static void beginInteractive(final InteractiveTokenCommand command) {
                         new Intent(CANCEL_INTERACTIVE_REQUEST)
                 );
             }
-
+            Logger.info(TAG,"Beginning execution with " + interactiveCommands.getAndIncrement() +
```

Comment:
I guess this comment can be misleading...I believe there would always ever be 1 of these interactive requests at any time as the thread pool size is 1

Reply:
The pool size is 1, but the queue is unbounded.